app/views/admin/users/show.html.haml

```diff
@@ -1,18 +1,44 @@
+= render 'admin/nav', {active: 'users'}
+
 %h1
   = @user.name
   - if current_user.has_role?(:admin)
     &nbsp;
     %small
       = link_to 'Edit', edit_admin_user_path(current_user), class: 'btn btn-default'
 
+- if @user.description.present?
+  %p
+    %b
+      Description:
+    %div
+      = @user.description
+
 %p
-  @user.description
+  %b
+    Admin?: <b>#{@user.has_role?(:admin)}</b>
 
 %p
-  Roles:
+  Dispatcher for Zones:
+  %ul
+    - if @dispatching_for.present?
+      - zones.each do |rz|
+        %li
+          = rz.name
+    - else
+      %li
+        %i
+          None
+
 
 %p
-  Driving for Ride Zones:
-  - RideZone.find_roles(:driver, @user).each do |rz|
-    %div
-      = rz.name
+  Driving for Zones:
+  %ul
+    - if @driving_for.present?
```

Comment:
Do we want to support multiple zones for drivers? If so, there is logic that needs to change around event notifications. Currently there is a User method driver_ride_zone_id that returns the first ID assuming there was a 1:1 of driver to ride zone. If we are going to allow drivers to drive in multiple zones that logic also needs to change (basically it would need to loop and broadcast events to multiple ride zones).